### ISSUE TITLE
Opening custom ontology that uses local imports

### ISSUE THREAD

**Issue by lsarni** [VisualDataWeb/WebVOWL#130]
I was trying to use the website with an ontology that imports other ontologies (custom ones, that aren't online yet) and it said `Loading` but it didn't display anything.

I downloaded the OWL2VOWL jar and try to create the .json. That was when I realize the problem was the imported ontologies:

```
org.semanticweb.owlapi.model.UnloadableImportException
Could not load imported ontology: <http://www.semanticweb.org/lsarni/ontologies/curricula-professors> Cause: OWLOntologyCreationIOException: java.io.FileNotFoundException: http://semanticweb.org/lsarni/ontologies/curricula-professors
```

After following the instructions on the README I managed to create a valid json and open it in the web.

It would be great if imports could be selected on the web or to display the error and some information to explaining how to work around it.

**Comment by vitalis-wiens** [comment]
Hi, thanks for the issue, 

It is more related to OWL2VOWL
and are currently working on the import issue, which should be fixed soon

**Comment by vitalis-wiens** [comment]
You are right, I will try to add more information output when an error occurs 

Also I will take a look into the - naming issue, thanks for the hint
